src/util.js

```diff
@@ -0,0 +1,13 @@
+import { Console } from '@woowacourse/mission-utils';
+
+export const errorString = (message) => `[ERROR] ${message}`;
```

Comment:
```suggestion
export const createErrorMessage = (message) => `[ERROR] ${message}`;
```
`errorString`라는 이름은 정적인 느낌있어서 유틸에서 사용하기는 조금 어색하다는 느낌이 들었는데 `createErrorMessage`와 같이 에러 메세지를 만든다는 의미는 어떨까요??